public/index.html

```diff
@@ -252,7 +252,7 @@ <h3>Example</h3>
 
       <p>Displaying documentation for example functions:</p>
 
-      <dl><dt><strong>formatName</strong></dt><dd>Formats a user's full name with proper capitalization</dd><dt>Parameter: <code>firstName</code></dt><dd>The user's first name</dd><dt>Parameter: <code>lastName</code></dt><dd>The user's last name</dd><dt>Parameter: <code>includeTitle</code></dt><dd>Whether to include a title prefix</dd><dt>Returns:</dt><dd>The formatted full name</dd></dl>
+      <dl><dt><strong>formatName</strong></dt><dd>Formats a user's full name with proper capitalization</dd><dt>Parameter: <code>firstName</code>: <span style="color: #0066cc;">string</span></dt><dd>The user's first name</dd><dt>Parameter: <code>lastName</code>: <span style="color: #0066cc;">string</span></dt><dd>The user's last name</dd><dt>Parameter: <code>includeTitle</code>: <span style="color: #0066cc;">boolean</span> (optional)</dt><dd>Whether to include a title prefix</dd><dt>Returns: <span style="color: #009900;">string</span></dt><dd>The formatted full name</dd></dl>
```

Comment:
The file `index.html` should be renamed to `index.template.html`. It is not meant to be served as is, but to be traasformed by the webpack `DocoffFunctionDocPlugin` plugin.

This plugin looks for the `<docoff-function-doc>` tag and replaces it with the `<dl>`. Hower now the `<dl>`  is  hardcoded here and that is wrong.

So in summery the `<dl>` on this line should not be in git as it must be dynamically generated at build time.